File: Updatingto14/Models/InvestedStock.cs
using System;
using System.Collections.Generic;

namespace Updatingto14.Models
{
    public partial class InvestedStock
    {
        public int Id { get; set; }
        public string? InvestedTicker { get; set; }
        public float? PurchasePrice { get; set; }
        public int? UserId { get; set; }
        public int? SharesOwned { get; set; }
        [System.Text.Json.Serialization.JsonIgnore]
        public virtual Stonk? InvestedTickerNavigation { get; set; }
        [System.Text.Json.Serialization.JsonIgnore]
        public virtual User? User { get; set; }
    }
}

File: Updatingto14/ClientApp/src/app/invested-stock.ts
import { Stonk } from "./stonk";
import { User } from "./user";

export interface InvestedStock {
    id: number;
    investedTicker: string;
    purchasePrice: number;
    userId: number;
    sharesOwned: number;
    investedTickerNavigation: Stonk;
    user: User;
}
